core/core/patch_test.go

```diff
@@ -1,15 +0,0 @@
-package core
-
-import (
-	"os"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-)
-
-// sets os.Stdout and os.Stderr to nil
-func TestSetsOsStdoutAndStderrToNil(t *testing.T) {
```

Comment:
@VaibhavMalik4187 it was your test that broke coverage... for still unknown reasons

Reply:
Ahh okay, I should've been more careful. Thanks for the fix.